exercises/grep/canonical-data.json

```diff
@@ -59,69 +59,83 @@
         {
           "description": "One file, one match, no flags",
           "property": "grep",
-          "pattern": "Agamemnon",
-          "flags": [],
-          "files": ["iliad.txt"],
+          "input": {
+              "pattern": "Agamemnon",
```

Comment:
I notice that this line is indented four spaces past the previous line, as opposed to other indentations which are two spaces from the previous line. Is there a reason why this should be so?

Reply:
No reason, I will fix this. Good catch.